cclib/bridge/cclib2pyscf.py

```diff
@@ -22,12 +24,55 @@ def _check_pyscf(found_pyscf):
 def makepyscf(atomcoords, atomnos, charge=0, mult=1):
     """Create a Pyscf Molecule."""
     _check_pyscf(_found_pyscf)
-    mol = gto.Mole(
-        atom = [['{}'.format(atomnos[i]),atomcoords[i]] for i in range(len(atomcoords))],
-        unit="Angstrom",
-        charge=charge,
-        multiplicity=mult
-    )
-    return  mol
-  
+    # See what can be parsed:

```

Comment:
Can you revert the changes to `makepyscf`? It looks like they aren't used right now.